provider/openstack/credentials.go

```diff
@@ -20,13 +21,15 @@ import (
 
 const (
 	CredAttrTenantName        = "tenant-name"
+	CredAttrTenantID          = "tenantID"
```

Comment:
Just noticed this. Should be "tenant-id" to be consistent with the other values.